packages/curriculum-parser-markdown/plugins/insight/enki-links.js

```diff
@@ -0,0 +1,51 @@
+const visit = require('unist-util-visit');
+const { sectionNames, internalLinks } = require('@enkidevs/curriculum-helpers');
+const UrlParse = require('url-parse');
+
+const { CONTEXTS } = internalLinks;
+
+module.exports = function enkiLink() {
+  return transform;
+
+  function transform(ast) {
+    const content = ast.children.find(
+      (node) =>
+        node.type &&
+        node.type === 'section' &&
+        node.name === sectionNames.CONTENT
```

Comment:
No need for this check:

```suggestion
        node.type === 'section' &&
        node.name === sectionNames.CONTENT
```

Reply:
right, cuz we validate